KarhooUISDK/Translations/Base.lproj/Localizable.strings

```diff
@@ -236,7 +236,7 @@
 "Text.Booking.PrebookConfirmed.RideDetails" = "Ride Details";
 
 /* Booking */
-"Text.Booking.RequestCar" = "Book";
+"Text.Booking.RequestCar" = "PAY AND BOOK";
```

Comment:
I am wondering if this is aligned with Android? Do we need to do it as part of this PR?

Check
https://github.com/karhoo/karhoo-android-ui-sdk/pull/256/files

Reply:
@dianakarh Yes, i do believe this string was updated on Android. I do remember updating the book button not too long ago

Reply:
Can you please double-check the A/Cs and make sure both platforms are aligned? It seems Android is different and this discrepancy does not make sense.

https://github.com/karhoo/karhoo-android-ui-sdk/blob/develop/uisdk/src/main/res/values/strings.xml#L33